gulpfile.mjs

```diff
@@ -7,21 +7,30 @@ import {
 } from '@hypothesis/frontend-build';
 import gulp from 'gulp';
 
+import tailwindConfig from './tailwind.config.mjs';
+
 gulp.task('build-js', () => buildJS('./rollup.config.mjs'));
 gulp.task('watch-js', () => watchJS('./rollup.config.mjs'));
 
 gulp.task('build-css', () =>
-  buildCSS([
-    './lms/static/styles/lms.scss',
-    './lms/static/styles/reports.css',
-    './lms/static/styles/frontend_apps.scss',
-    './lms/static/styles/ui-playground/ui-playground.scss',
-  ])
+  buildCSS(
+    [
+      './lms/static/styles/lms.scss',
+      './lms/static/styles/reports.css',
+      './lms/static/styles/frontend_apps.scss',
+      './lms/static/styles/ui-playground/ui-playground.scss',
+    ],
+    { tailwindConfig }
+  )
 );
 
 gulp.task('watch-css', () => {
   gulp.watch(
-    './lms/static/styles/**/*.{css,scss}',
+    [
+      './lms/static/styles/**/*.{css,scss}',
+      './lms/static/scripts/frontend_apps/**/*.js',
+      './lms/static/scripts/ui-playground/**/*.js',
```

Comment:
With tailwind in purge mode, we'll need to rebuild the CSS when component source changes, to get the utility classes used therein included in output CSS